tags/faq/llm.ytag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+LLM is bad and inconsistent at writing code. Do not use ChatGPT or other large language models to try to write mods. Please learn java first.
```

Comment:
> LLM is bad

I disagree, its a great tool if used right. Sadly people expect it to just write a whole mod in one go. I think we should have a tag that explains why it might not be writing perfect code for them.